typescript/infra/config/environments/mainnet/funding.ts

```diff
@@ -13,5 +14,7 @@ export const relayerFunderConfig: RelayerFunderConfig = {
   prometheusPushGateway:
     'http://prometheus-pushgateway.monitoring.svc.cluster.local:9091',
   contextFundingFrom: Contexts.Abacus,
-  contextsToFund: [Contexts.Abacus],
+  contextsAndRolesToFund: {
+    [Contexts.Abacus]: [KEY_ROLE_ENUM.Relayer],
```

Comment:
Is the role here just to give debug info or do you see it making changes at runtime?